Readme: Make init script executable (needed for linux)

Garandor opened the pull request:
Alternatives would be to invoke as 
> . ./scripts/init.sh 
or 
> source ./scripts/init.sh

No functional changes



Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (`manta` or `dolphin`) with right title (start with [Manta] or [Dolphin]),
- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests.
- [ ] Updated relevant documentation in the code.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
- [ ] If needed, notify the committer this is a draft-release and a tag is needed after merging the PR.
- [ ] Verify benchmarks & weights have been updated for any modified runtime logics
- [ ] If needed, bump `version` for every crate.
- [ ] If import a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- [ ] If needed, update our Javascript/Typescript APIs. These APIs are offcially used by exchanges or community developers.
- [ ] If we're going to issue a new release, freeze the code one week early(it depends, but usually it's one week), ensure we have enough time for related testing.

Commit:
Make init script executable

Alternatives would be to invoke as 
> . ./scripts/init.sh 
or 
> source ./scripts.init.sh

ghzlatarev commented:
Yo @Garandor you need at least @Dengjianping or @stechu as reviewers as well as they have the merge privileges.

ghzlatarev commented on README.md:
```diff
@@ -72,4 +73,4 @@ now you can watch the ci build your awesome feature and publish your draft relea
 
 ## Minimum supported rust compiler
 
-This project's MSRV is `rustc 1.56`
```
why the extra line

Garandor replied:
Was autoadded by Github's integrated markdown editor (it's probably part of the MD spec to end files in a newline)
anyway, i got rid of it

ghzlatarev commented:
> Yo @Garandor you need at least @Dengjianping or @stechu as reviewers as well as they have the merge privileges.

Otherwise seems fine.

Commit:
Update README.md

fix the autoadded newline

Commit:
Update README.md

One more try, it looks like github's integrated markdown editor likes to put newlines at the end of .mds on commit

Commit:
Manually remove the newline...

Commit:
and readd the removed one...

Garandor commented:
@Dengjianping I retargeted this against 3.1.2 - it's only a markdown change
